ui/oopsyraidsy/oopsy_live_list.ts

```diff
@@ -272,9 +272,11 @@ export class OopsyLiveList implements MistakeObserver {
 
     const iconClass = m.type;
     const blame = m.name ?? m.blame;
-    const blameText = blame ? ShortNamify(blame, this.options.PlayerNicks) + ': ' : '';
+    const blameText = (blame !== undefined && blame.length > 0)
```

Comment:
I realize this is not what's there before, but I think it's weird logic to skip the name because the name is blank.  What about the blank name turning into `???` and undefined using `''` instead of `': '`?